test/TesterInternal/HostedTestClusterBase.cs

```diff
@@ -27,33 +27,4 @@ public virtual void Dispose()
             HostedCluster?.StopAllSilos();
         }
     }
-
-    public abstract class HostedTestClusterPerTest : OrleansTestingBase, IDisposable
-    {
-        protected TestCluster HostedCluster { get; private set; }
-
-        public HostedTestClusterPerTest()
-        {
-            TestDefaultConfiguration.InitializeDefaults();
-
```

Comment:
So TestDefaultConfiguration.InitializeDefaults(); is needed? then why do we remove the one duplicate which has it ?